DirectProgramming/C++SYCL_FPGA/Tutorials/Features/IPAuthoring_interfaces_overview/Readme.md

```diff
@@ -0,0 +1,255 @@
+>
+> For using the simulator flow, Intel® Quartus® Prime Pro Edition and one of the following simulators must be installed and accessible through your PATH:
+> - Questa*-Intel® FPGA Edition
+> - Questa*-Intel® FPGA Starter Edition
+> - ModelSim® SE
+>
+> When using the hardware compile flow, Intel® Quartus® Prime Pro Edition must be installed and accessible through your PATH.
+
+> **Warning**: Make sure you add the device files associated with the FPGA that you are targeting to your Intel® Quartus® Prime installation.
+
+This sample is part of the FPGA code samples.
+It is categorized as a Tier 1 sample that helps you getting started.
+```mermaid
+flowchart LR
+   tier1("Tier 1: Get Started")
+   tier2("Tier 2: Explore the Fundamentals")
+   tier3("Tier 3: Explore the Advanced Techniques")
+   tier4("Tier 4: Explore the Reference Designs")
+
+   tier1 --> tier2 --> tier3 --> tier4
+
+   style tier1 fill:#f96,stroke:#333,stroke-width:1px,color:#fff
+   style tier2 fill:#0071c1,stroke:#0071c1,stroke-width:1px,color:#fff
+   style tier3 fill:#0071c1,stroke:#0071c1,stroke-width:1px,color:#fff
+   style tier4 fill:#0071c1,stroke:#0071c1,stroke-width:1px,color:#fff
+```
+
+Find more information about how to navigate this part of the code samples in the [FPGA top-level README.md](/DirectProgramming/C++SYCL_FPGA/README.md).
+You can also find more information about [troubleshooting build errors](/DirectProgramming/C++SYCL_FPGA/README.md#troubleshooting), [running the sample on the Intel® DevCloud](/DirectProgramming/C++SYCL_FPGA/README.md#build-and-run-the-samples-on-intel-devcloud-optional), [using Visual Studio Code with the code samples](/DirectProgramming/C++SYCL_FPGA/README.md#use-visual-studio-code-vs-code-optional), [links to selected documentation](/DirectProgramming/C++SYCL_FPGA/README.md#documentation), and more.
+
+## Invocation Interface
+The Intel icpx compiler generates an interface for your SYCL* kernel. The invocation interface controls how the component communicates with the rest of your system. This interface lets your wider system start the component, and lets the component inform the system when it is finished.
+
+### MM Agent
+The signals for starting your component, and the signals used by the component to signal its status are stored in a MM Agent space. You should use an MM Agent invocation interface if your component will sit in a system like this: \
+![](Readme.resource/invocation_mm_agent.png)
+
+By default, the Intel icpx compiler generate MM Agent invocation interface for kernels, as you may observed in the [Naive Design](naive).
+
+### Streaming
+The signals for starting your component, and the signals used by the component to signal its status are stored in a MM Agent space. You should use an MM Agent invocation interface if your component will sit in a system like this: \
```

Comment:
I wonder if this is correct? It is identical to MM Agent description above and mentioning  "You should use an **_MM Agent_** invocation ..."

Reply:
you are correct, this is a mistake